imgProcess.py

```diff
@@ -0,0 +1,72 @@
+# Tayo Elelu
+# Created 10/12/16
+# CS 4365 - Software Engr II
+# Image Classification Module
+
+# This module is used to generalize image classification
+# for the project. Ultimately, it should be able to take in a
+# group of images and classify them to a specified target, and print results.
+
+# 1 - Group of Images to CSV file Script
+# 2 - Apply svm.svc to CSV of images
+
+# PROBLEM - to use Scikit for image processing, number of pixels
+# must be the same in each image, so padding/resizing/reshaping is
+# needed. Drastically limits the generality of a module.
+
+##############################################################
+
+# Standard scientific Python imports
+import matplotlib
+matplotlib.use('Agg')         #Force matplotlib to not use any Xwindows backend (causes errors)
+import matplotlib.pyplot as plt
+
+# Import datasets, classifiers and performance metrics
+from sklearn import datasets, svm, metrics
+
+# The digits dataset
+digits = datasets.load_digits()

```

Comment:
Yeah doing the testing on classification itself is easier, its harder to come up with test cases for the processFolder module, which is what the bulk of my focus has been on